model_draft_transactions_test.go

```diff
@@ -62,7 +64,9 @@ func TestDraftTransaction_GetModelName(t *testing.T) {
 	t.Parallel()
 
 	t.Run("model name", func(t *testing.T) {
-		draftTx := newDraftTransaction(testXPub, &TransactionConfig{}, New())
+		draftTx := newDraftTransaction(testXPub, &TransactionConfig{
+			FeeUnit: chainstate.MockDefaultFee,
```

Comment:
Isn't it set by test config of bux?

Reply:
Normally it is, but in this test case the `newDraftTransaction` function is tested in isolation. So the `draft.Client()` which we talked about will return nil if we don't provide FeeUnit by `TransactionConfig`.

Reply:
Let's leave it like it is for now.
But for the future, for me it is very weird, that we're testing function without "client", and actually this function partially depends on this client.